flocker/restapi/_infrastructure.py

```diff
@@ -223,7 +223,7 @@ def got_result(result):
     return deco
 
 
-def user_documentation(doc, examples=None):
+def user_documentation(doc, examples=None, header=None):
```

Comment:
If it is a problem that there isn't a header, then it probably makes sent to require it to be present, rather than having a default value for it.